.github/workflows/build-test-actions.yml

```diff
@@ -44,8 +39,11 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install poetry
-          poetry install --without docs --no-interaction
-
+          poetry install --with lint --without docs --no-interaction
```

Comment:
there's no need to add `with lint`. everything besides doc dependencies gets installed.